assets/js/pages/project/create.vue

```diff
@@ -58,12 +72,27 @@
             async createProject() {
                 this.$v.$touch()
                 if (!this.$v.$invalid) {
-                    await api.create(this.project.name, this.project.description)
+                    await api.create(this.project.id, this.project.name, this.project.description)
                     this.$router.push('/')
                 }
             }
         },
+        watch: {
+            'project.id': function (newValue, oldValue) {
```

Comment:
If we are not using oldValue, it's not necessary to put it on argument list, it's javascript overall ;)

Reply:
Indeed I was too haste, I added both when I was checking how function works and forgot to cleanup.